folsom/src/main/java/com/spotify/folsom/client/ascii/SetRequest.java

```diff
@@ -40,39 +41,46 @@
     CMD.put(Operation.APPEND, "append ".getBytes(US_ASCII));
     CMD.put(Operation.PREPEND, "prepend ".getBytes(US_ASCII));
     CMD.put(Operation.CAS, "cas ".getBytes(US_ASCII));
-  }
 
-  private static final byte[] FLAGS = " 0 ".getBytes(US_ASCII);
+    NO_FLAGS = "0".getBytes(US_ASCII);
```

Comment:
I don't think this needs to go in the `static {}` block.

Reply:
Not really - I will move it to the same line as the declaration.